### PR TITLE
Migrate compute canada instructions to sphinx doc

### PR DESCRIPTION
**Pull request by lpsaavedra**
# Description of the problem

The instructions to run Lethe on Compute Canada clusters was only available on the wiki.

# Description of the solution

This PR migrates the instructions to the new sphinx documentation.

# How Has This Been Tested?

This does not require any test. 

# Future changes

None

# Comments

None

### CODE REVIEW

**Review by blaisb — APPROVED** [review]
Wow awesome, this is very clear.

File: doc/source/compute_canada.rst [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+ module load parmetis/4.0.3
+ module load p4est/2.0
+
+Then, we can clone and compile ``dealii``. Although Lethe always supports the master branch of deal.II, we maintain an identical deal.II fork on the lethe repository. This fork is always tested to make sure it works with lethe. To clone the deal.II fork github repository, execute in ``$HOME/dealii`` directory:
+
+.. code-block:: text
+
+ git clone https://github.com/lethe-cfd/dealii.git
+
+We can compile ``dealii`` in the ``$HOME/dealii/build folder``, by defining path to installation folders of ``Trilinos``, ``Parmetis`` and ``P4est``:
```
Comment (oguevremont) [inline doc/source/compute_canada.rst:58]:
```suggestion
We can compile ``dealii`` in the ``$HOME/dealii/build folder``, by defining the paths to installation folders of ``Trilinos``, ``Parmetis`` and ``P4est``:
```

File: doc/source/compute_canada.rst [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+ module load parmetis/4.0.3
+ module load p4est/2.2
+
+Then, we can clone and compile ``dealii``. Although Lethe always supports the master branch of deal.II, we maintain an identical deal.II fork on the lethe repository. This fork is always tested to make sure it works with lethe. To clone the deal.II fork github repository, execute in ``$HOME/dealii`` directory:
+
+.. code-block:: text
+
+ git clone https://github.com/lethe-cfd/dealii.git
+
+We can compile ``dealii`` in the ``$HOME/dealii/build`` folder, by defining path to installation folders of ``Trilinos``, ``Parmetis`` and ``P4est``:
```
Comment (oguevremont) [inline doc/source/compute_canada.rst:95]:
```suggestion
We can compile ``dealii`` in the ``$HOME/dealii/build`` folder, by defining the paths to installation folders of ``Trilinos``, ``Parmetis`` and ``P4est``:
```

File: doc/source/compute_canada.rst [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+
+ source $HOME/.dealii
+
+and use it in your ``.sh`` script (see Launching simulations below).
+
+================================
+Launching simulations
+================================
+
+Simulations are sent to the scheduler via batch scripts, visit Compute Canada wiki page more information about the `scheduler <https://docs.computecanada.ca/wiki/What_is_a_scheduler%3F>`_ and `running jobs <https://docs.computecanada.ca/wiki/Running_jobs>`_. For your convenience, an example of ``job.sh`` used on Beluga is given below:
```
Comment (oguevremont) [inline doc/source/compute_canada.rst:219]:
```suggestion
Simulations are sent to the scheduler via batch scripts. Visit the Compute Canada wiki page for more information about the `scheduler <https://docs.computecanada.ca/wiki/What_is_a_scheduler%3F>`_ and `running jobs <https://docs.computecanada.ca/wiki/Running_jobs>`_. For your convenience, an example of ``job.sh`` used on Beluga is given below:
```

**Review by oguevremont — APPROVED** [review]
Very clear!

**Pull request merged**